extensions/libraries/redcore/form/base.php

```diff
@@ -73,80 +74,6 @@ public static function getInstance($name, $data = null, $options = array(), $rep
 	}
 
 	/**
-	 * Override the validate() method to store the error per field.
-	 *
-	 * @param   array   $data   An array of field values to validate.
-	 * @param   string  $group  The optional dot-separated form group path on which to filter the
-	 *                          fields to be validated.
-	 *
-	 * @return  mixed  True on sucess.
-	 */
-	public function validate($data, $group = null)
```

Comment:
I checked that function and that function exists both in j3 and j25 that is why I removed it here... the difference in versions is minor and should work for both... Do you have an example or case where it would not work @phproberto